install_config/persistent_storage/persistent_storage_fibre_channel.adoc

```diff
@@ -51,34 +40,29 @@ spec:
   accessModes:
     - ReadWriteOnce
   fc:
-    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <1>
-    lun: 2
+    wwids: [scsi-3600508b400105e210000900000490000] <1>
+    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <2>
+    lun: 2 <2>
     fsType: ext4
 ----
-<1> Fibre Channel WWNs are identified as `/dev/disk/by-path/pci-<IDENTIFIER>-fc-0x<WWN>-lun-<LUN#>`, but you do not need to provide any part of the path leading up to the `WWN`, including the `0x`, and anything after, including the `-` (hyphen).
+<1> Optional: World wide identifiers (WWIDs). Either FC `*wwids*` or a combination of FC `*targetWWNs*` and `*lun*` must be set, but not both simultaneously. The FC WWID identifier is recommended over the WWNs target because it is guaranteed to be unique for every storage device, and independent of the path that is used to access the device. The WWID identifier can be obtained by issuing a SCSI Inquiry to retrieve the Device Identification Vital Product Data (`page 0x83`) or Unit Serial Number (`page 0x80`). FC WWIDs are identified as `/dev/disk/by-id/` to reference the data on the disk, even if the path to the device changes and even when accessing the device from different systems.
+<2> Optional: World wide names (WWNs). Either FC `*wwids*` or a combination of FC `*targetWWNs*` and `*lun*` must be set, but not both simultaneously. The FC WWID identifier is recommended over the WWNs target because it is guaranteed to be unique for every storage device, and independent of the path that is used to access the device. FC WWNs are identified as `/dev/disk/by-path/pci-<IDENTIFIER>-fc-0x<WWN>-lun-<LUN#>`, but you do not need to provide any part of the path leading up to the `WWN`, including the `0x`, and anything after, including the `-` (hyphen).
```

Comment:
Wondering whether the use of uppercase in the replaceables (`/dev/disk/by-path/pci-<IDENTIFIER>-fc-0x<WWN>-lun-<LUN#>`) is necessary, or can be lowercased. I see it predates the PR though.

Also: s/leading up to the `WWN`/leading up to the `<WWN>`/ ?

Also also: I see mostly the file sticks with the existing 3.x-era backtick+asterisk style of markup, though the `/dev/disk/by-path...` one is the only one using plain backticks. :shrug:

Reply:
Thanks, @adellape - initially, I wasn't touching existing values because of my lack of familiarity and since they were already there. But there are places out of this PR that I did choose to fix while the ones you pointed out didn't get touched. (I also couldn't recall what we had for markup rules in 3.11.) I agree with all your points, though, and I've updated for consistency in this doc.